CMakeLists.txt

```diff
@@ -208,14 +208,26 @@ endif(MIR_DISABLE_EPOLL_REACTOR)
 
 add_definitions(-DMESA_EGL_NO_X11_HEADERS)
 
-# Default to KMS backend, but build all of them
-set(
-  MIR_PLATFORM
-  mesa-kms;mesa-x11;eglstream-kms
-  CACHE
-  STRING
-  "a list of graphics backends to build (options are 'mesa-kms', 'mesa-x11', or 'eglstream-kms')"
-)
+find_library(NVIDIA_EGL_WAYLAND NAMES nvidia-egl-wayland libnvidia-egl-wayland.so.1 libnvidia-egl-wayland.so.1.0.2 libnvidia-egl-wayland.so.1.0.3)
```

Comment:
Can't we just use the pkgconfig names here? `wayland-eglstream` and `wayland-eglstream-protocols` are the pkgconfig module names.

Reply:
That would be nice, but AIUI the Ubuntu packages are somewhat behind Fedora.

Of course, we should try pkgconfig first.